### ISSUE TITLE
Different loss calculation in code and paper

### ISSUE THREAD

**Issue by chansonzhang** [suragnair/alpha-zero-general#329]
In [code](https://github.com/suragnair/alpha-zero-general/blob/f1a78e0505c614205f8c60eaa82cd644a07a84b3/othello/pytorch/NNet.py#L97):

```Python
def loss_pi(self, targets, outputs):
        return -torch.sum(targets * outputs) / targets.size()[0]
```

In [paper](https://github.com/suragnair/alpha-zero-general/blob/master/pretrained_models/writeup.pdf)

![Image](https://github.com/user-attachments/assets/43aa2663-0ca5-47e9-8b90-e3f86a1242a0)

Shouldn't it be - instead of +? Is this a typo in the paper?

**Comment by suragnair** [comment]
Looks like a typo in the paper! Thanks for spotting it.
